docs/ci_runner_setup/linux/runner_setup_1.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# runner setup
+mkdir "actions-runner-$1" && cd "actions-runner-$1"
+curl -o actions-runner-linux-x64-2.323.0.tar.gz -L https://github.com/actions/runner/releases/download/v2.323.0/actions-runner-linux-x64-2.323.0.tar.gz
```

Comment:
Can you make the runner version an argument to the script, or do something to select the latest?